css/manual.css

```diff
@@ -43,18 +44,9 @@ h3 {
     color: inherit;
 }
 
-#content {
-    width: 1170px;
-    margin-right: auto;
-    margin-left: auto;
-    margin-top: -23px;
-    padding-bottom: 10px;
-    padding-left: 15px;
-    padding-right: 15px;
-    -webkit-border-radius: 5px;
-    -moz-border-radius: 5px;
+pre.src {
+    overflow-x: auto;
```

Comment:
I always forget this trick, to avoid having scrollbar when it's not required. And use scrollbar only along x.